src/benchmarks/micro/libraries/System.IO/StreamReaderReadLineTests.cs

```diff
@@ -12,28 +12,34 @@ namespace System.IO.Tests
     [BenchmarkCategory(Categories.Libraries)]
     public class StreamReaderReadLineTests : TextReaderReadLineTests
     {
-        private byte[] _bytes;
+        private StreamReader _reader;
 
         [GlobalSetup]
         public void GlobalSetup()
         {
             _text = GenerateLinesText(LineLengthRange, 16 * 1024);
-            _bytes = Encoding.UTF8.GetBytes(_text);
+            _reader = new (new MemoryStream(Encoding.UTF8.GetBytes(_text)));
```

Comment:
```suggestion
            _reader = new StreamReader(new MemoryStream(Encoding.UTF8.GetBytes(_text)));
```
I'd write the type explicit here, as the declaration is "too far away".

Reply:
Done